vic/drivers/shared_image/src/vic_init.c

```diff
@@ -1106,7 +1107,9 @@ vic_init(void)
         }
         // handle the bare soil portion of the tile
         vidx = veg_con_map[i].vidx[options.NVEGTYPES - 1];
-        Cv_sum[i] += veg_con[i][vidx].Cv;
+        if (vidx != NODATA_VEG) {
+            Cv_sum[i] += veg_con[i][vidx].Cv;
+        }
```

Comment:
The subsequent block dealing with veg above the treeline is wrong, I think.  Why is it using options.NVEGTYPES - 1 as the index of the tile in this particular grid cell pertaining to above-treeline veg?  Doesn't make sense.  options.NVEGTYPES is a global variable - it's *not* the number of non-zero area tiles in this particular grid cell.  You need to use `veg_con[i][vidx].vegetat_type_num` as the index (with or without a -1).  This is likely a source of runtime errors, since most grid cells will not, in general, have as many tiles as there are classes in the library, so attempting to access the vidx of options.NVEGTYPES (with or without a -1) will give a vidx of -1 I think, which you can't use as an index...  There may be other problems with compute treeline but that's a start.